content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,24 @@
+title: "A Statement on Richard Stallman and the Free Software Foundation"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+_A Statement from Mixxx on Richard Stallman's return to the FSF._
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.
+His resignation came as a relief for many who have suffered his bad behavior.
+Richard Stallman's actions disqualify him from representing the FSF and the free software community as a whole.
```

Comment:
```suggestion
```
We can't prove this statement and it should be clear anyway from the sentences before.

Reply:
Whether someone is qualified for something is an opinion, not a fact. We can voice our opinion here.

Reply:
Yes, but we have decided to do a blog post and not sign as org, to take the chance to explain the situation regarding to Mixxx like other orgs did.

It does not work well to express an opinion when you speak for a whole group.

Reply:
I don't follow your reasoning. You seem to imply that you agree with the sentence but we shouldn't publish it unless we all agree? But don't we all agree on this?

Reply:
It's redundant to say "we believe" in a blog post about our beliefs. I can add it back but it's unnecessary verbiage because the existence of the blog post implies this is our belief as a group.

Reply:
It's redundant but it doesn't hurt either. If it helps so that we can agree on this, let's just add it.